### PR TITLE
added documentation on pyxform

### LINKED ISSUE #7: pyxform: describe how to get the command-line tool
`pyxform` is used as an underlying library in several form-conversion tools.

For some people, the command-line tool can be convenient and it's easy to install with `pip install pyxform`.

https://github.com/XLSForm/pyxform

### PR DESCRIPTION
**Pull request by Divya063**
closes #7 

## What is included in this PR?
Documentation on pyxform 

## What new work does this PR create?
A suitable place has to be found for this doc

### CODE REVIEW

File: pyxform.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  
+The :command:`xls2xform` command can then be used:
+
+.. code-block:: console
+  
+  $ xls2xform path_to_XLSForm output_path
+  
+.. tip::
+  
+  For futher queries or more information please refer to the `Github repo <https://github.com/XLSForm/pyxform>`_.
```
Comment (lognaturel) [inline pyxform.rst]: futher -> further! I would just say "for more information", though.

File: pyxform.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+pyxform
+************
+
+**pyxform** is a Python library that makes writing XForms for ODK Collect and enketo easy by converting XLS(X) spreadsheets into XForms. It is used as a library in a number of tools including the `ODK online converter <http://opendatakit.org/xiframe/>`_ and `Ona <https://ona.io/>`_.
+
+.. _runningpyxform:
+
+Running pyxform from command line
+====================================
+For those who want to convert forms at the command line ,pyxform can be installed directly from the command line using `pip <https://en.wikipedia.org/wiki/Pip_(package_manager)>`_:
```
Comment (lognaturel) [inline pyxform.rst]: "line ,pyxform" -> "line, pyxform"

File: pyxform.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
+************
+pyxform
+************
+
+**pyxform** is a Python library that makes writing XForms for ODK Collect and enketo easy by converting XLS(X) spreadsheets into XForms. It is used as a library in a number of tools including the `ODK online converter <http://opendatakit.org/xiframe/>`_ and `Ona <https://ona.io/>`_.
+
+.. _runningpyxform:
```
Comment (adammichaelwood) [inline pyxform.rst]: hyphenate section labels: 
running-pyxform

**Comment by adammichaelwood** [comment]
> A suitable place has to be found for this doc

I think where you put it (Form Design) is totally reasonable.
We *might* in the future create a "Tools and Utilities" section, but for now this is fine.

**Comment by Divya063** [comment]
Thanks for the review! I applied the requested changes.

**Comment by adammichaelwood** [comment]
Great. Thanks!

**Pull request merged**